sample/www/jxcore/thali/mockmobile.js

```diff
@@ -0,0 +1,58 @@
+var keys = {};
+
+function Mobile(key) {
+  keys[key] || (keys[key] = new NativeCall(key));
+  return keys[key];
+}
+
+Mobile.invokeNative = function (key, arg) {
+  keys[key].registerNativeCallback(arg);
```

Comment:
A little documentation would be nice. Specifically, what is a key and what methods is it supposed to have?